src/bilder/images/edx_notes/deploy.py

```diff
@@ -102,30 +127,66 @@
         sink=[VaultAutoAuthSink(type="file", config=[VaultAutoAuthFileSink()])],
     ),
 )
+vault = Vault(
+    versions=VERSIONS["vault"],
+    configuration={Path("vault.json"): vault_config},
+)
```

Comment:
This shouldn't be necessary because it is placed during the build of the docker_baseline image that this is using as the base AMI.